app/assets/javascripts/active_admin/qiniu_input/upload_video.js

```diff
@@ -72,6 +70,9 @@ $(document).ready(function() {
     formData = new FormData();
     formData.append('token', token);
     formData.append('file', f);
+    if (f.name && f.name.length > 0) {
+      formData.append('key', '' + (new Date()).getTime() + '-' + Math.floor(Math.random() * Math.floor(1000)) + '-' + f.name);
```

Comment:
use a variable for (new Date()).getTime() + '-' + Math.floor(Math.random() * Math.floor(1000))